website/docusaurus.config.js

```diff
@@ -272,7 +272,7 @@ const siteConfig = {
 
   scripts: [
     {
-      src: "https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js",
+      src: "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.min.js",
```

Comment:
Can we import the `jquery` from `unpkg.com`, like we handle other resources? I don't have strong preference over specific CDN, either cloudfare or unpkg look good to me. However it would be great to stick with one consistently: so that we don't have two external dependencies that could effectively break the website.